<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>531</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>686</width>
    <height>531</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>686</width>
    <height>531</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stepper Contoller for 282.478 Project II </string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>561</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Stepper Modes</string>
   </property>
   <widget class="QRadioButton" name="wav">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Wave Stepping</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="ful">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Stepping</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hlf">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Half Stepping</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mic">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Micro Stepping</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>551</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Rotation Direction</string>
   </property>
   <widget class="QRadioButton" name="cw">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Clockwise</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="ccw">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Counter Clockwsie</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>601</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Speed Input</string>
   </property>
   <widget class="QSlider" name="slider_speed">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>481</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>601</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Steps  Input</string>
   </property>
   <widget class="QSlider" name="slider_steps">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>481</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_steps">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>100 steps</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="conn_test">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>startClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>speedChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_steps</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>stepsChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>conn_test</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>tested()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startClicked()</slot>
  <slot>speedChanged(int)</slot>
  <slot>stepsChanged(int)</slot>
  <slot>tested()</slot>
 </slots>
</ui>
